test/export/test_converter.py

```diff
@@ -869,6 +936,22 @@ def forward(self, x):
         #         )[0], 1
         #     )
 
```

Comment:
maybe turn this into a parametrized test and merge these two tests?

Reply:
hmm, I tried 

jit.script() failed at `func(x_len, y_len)`

I guess scripting doesn't support capturing callable type.